locales/ja/json.json

```diff
@@ -387,7 +387,8 @@
     "Log In": "ログイン",
     "Log Out": "ログアウト",
     "Log out": "ログアウト",
-    "Log Out Other Browser Sessions": "全ての端末からログアウト",
+    "Log Out Other Browser Sessions": "全ての端末をログアウト",
+    "Log Out of Other Browser Sessions": "全ての端末からログアウト",
```

Comment:
No new keys need to be added.

Reply:
@andrey-helldar 

Hi!

Thanks for reviewing my PR.
Sorry for making new keys.....
deleted it, so please review again !

Reply:
In this project we do not create keys ourselves. The list of keys for translations fully corresponds to the available translations of all official Laravel products.

Reply:
@andrey-helldar 

Ok. I got it.
Thanks for replying for my proposal ! 🙌